glibc.yaml

```diff
@@ -113,6 +115,12 @@ pipeline:
   - runs: |
       make -C build -j$(nproc) install DESTDIR="${{targets.destdir}}"
 
+  - name: "Run the glibc testsuite, not blocking on failures for now"
+    runs: |
+      # For now we run the glibc testsuite for manual investigation.
+      # Later on we'll establish a baseline and make regressions fail the build.
+      make -C build -j$(nproc) check || true
```

Comment:
can we please store the testsuite.logs and package them into an apk? can be into -dev or a new one like -testresults?

As the builds are long, and it is customary to submit those to upstream with bug reports. Or just in general when testing RC builds.

Similarly for gcc, if we start running test-suite there, package it such that one can email it to upstream to submit test results.

Reply:
I like that idea. Let me create a `glibc-testresults` package for this!